develop.yml

```diff
@@ -50,3 +54,270 @@ services:
     volumes:
       - ./images/:/images/
       - ./cloud-upload-apis/:/cloud-upload-apis/
+  # # JITSI
+  # Frontend
+  web:
+      image: jitsi/web:stable-5963
+      restart: ${RESTART_POLICY}
+      ports:
+          - '${HTTP_PORT}:80'
+          - '${HTTPS_PORT}:443'
+      volumes:
+          - ${CONFIG}/web:/config:Z
+          - ${CONFIG}/transcripts:/usr/share/jitsi-meet/transcripts:Z
+      environment:
+          - ENABLE_COLIBRI_WEBSOCKET
+          - ENABLE_FLOC
+          - ENABLE_LETSENCRYPT
+          - ENABLE_HTTP_REDIRECT
+          - ENABLE_HSTS
+          - ENABLE_XMPP_WEBSOCKET
+          - DISABLE_HTTPS
```

Comment:
@Vedant1202 If this does what I think it does then it should probably be present only in develop.yml and not in camicroscope.yml

Reply:
Okay yeah, we can discuss about this